templates/css-variables.twig

```diff
@@ -1,7 +1,7 @@
-<style>
+<style id="test">
 	:root {
 		{% for key, value in css_vars %}
-			--{{ key }}: {{ value }};
+			--{{ key|trim('--', 'left') }}: {{ value|raw }};
```

Comment:
The existing implementation in the sidebar stored the variables without `--` at the front. For now, in the newer one `--` is included. Trimming here to make both work.

Reply:
The `|raw` filter actually solves an issue production currently has, if there are characters in the value that Twig escapes. Could be there are no such values in the old theme options, but in the new ones I already ran into it.

Reply:
👍